views/fe-utils/siteTooltipContent.js

```diff
@@ -0,0 +1,30 @@
+import { TOTAL_LABEL, NA } from '../../constants/vars'
+import { formatAsPercentage } from './formatAsPercentageUtil'
+
+export const siteTooltipContent = (siteData) => {
+  const tooltipData = []
+  let siteTotalCount = 0
+  let siteTotalPercent = 0
+
+  siteData
+    .filter(({ name }) => name !== NA)
```

Comment:
let's update the other places where we use the `N_A` and `TOTALS` constants as well.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/191/commits/32872dc7c65af9e91a8eaeffc68e80a7ca35bd2f

https://github.com/AMP-SCZ/dpdash/pull/191/commits/e9d38f2a91f03adbc64f06a73491bac4ad230b82